projects/igniteui-angular/src/lib/grids/cell.component.ts

```diff
@@ -458,6 +462,18 @@ export class IgxGridCellComponent implements OnInit, OnChanges, OnDestroy {
         this.grid.notifyChanges();
     }
 
+    /**
```

Comment:
Do we really need this? I suppose that the selected style would be the same for the whole component. We can just move the following check above in the `selected` getter:
```typescript
   return this.selectionService.isColumnSelected(this.column.field);
```